src/cudamatrix/cu-kernels-ansi.h

```diff
@@ -30,6 +30,9 @@
 #include "cudamatrix/cu-matrixdim.h"
 
 #if HAVE_CUDA == 1
```

Comment:
sure, that makes sense. I was wondering why we need to keep the same interface as cublas_copy before. And I just removed the lines adding this header (<cublas_v2.h>).
Ke